src/modules/idmef/PacketIDMEFReporterCfg.cpp

```diff
@@ -23,7 +23,8 @@ PacketIDMEFReporterCfg::PacketIDMEFReporterCfg(XMLElement* elem)
 		XMLElement* e = *it;
 
 		if (e->matches("snapshotoffset")) {
```

Comment:
Should we just remove the complete if statement?

Reply:
Only if we remove snapshotoffset from the XMLs as well. Otherwise it will cause MSG_FATAL down on the last else branch.

Reply:
`snapshotoffset`  does not have any effect anymore, does it? If there is no effect anymore, we should remove it from the `if` and also from the XML.